dotnet/src/IntegrationTests/Connectors/MistralAI/ChatCompletion/MistralAIChatCompletionTests.cs

```diff
@@ -70,10 +70,10 @@ public async Task ValidateInvokeChatPromptAsync()
             .AddMistralChatCompletion(model!, apiKey!)
             .Build();
 
-        const string ChatPrompt = @"
+        const string ChatPrompt = """
             <message role=""system"">Respond in French.</message>
             <message role=""user"">What is the best French cheese?</message>
-        ";
+        """;
```

Comment:
Using the new C# 12 literal, you don't need to escape `"` quotes anymore
```suggestion
        const string ChatPrompt = """
            <message role="system">Respond in French.</message>
            <message role="user">What is the best French cheese?</message>
        """;
```